lonabot/bot.py

```diff
@@ -77,6 +77,10 @@ async def wrapped(*args, **kwargs):
 `/remind 17:05 Optional text`
 `/remind 17/11/2020 20:00 Optional text`
 `/remind 2020-02-02T20:00:00+02:00 Optional text`
+`/remind later Optional text`
+
+`/remind later` requires you to first configure what `later` is with
+`/later 1y2mo3w4d5h6s`
```

Comment:
I see `1y2mo3w4d5h6s` and I get scared. Could this example be simpler? Maybe "5m" would be enough of an example?

Reply:
Has been simplified now.